README.md

```diff
@@ -154,7 +151,12 @@ Header-based and backend-independent Device API can be called within ```sycl ker
 
 ### Supported Configurations:
 
-Supported domains: BLAS, LAPACK, RNG, DFT, SPARSE_BLAS
+Supported domains include: BLAS, LAPACK, RNG, DFT, SPARSE_BLAS
+
+Supported compilers include:
+- [Intel(R) oneAPI DPC++ Compiler](https://software.intel.com/en-us/oneapi/dpc-compiler): Intel proprietary compiler that supports CPUs and Intel GPUs. Intel(R) oneAPI DPC++ Compiler will be referred to as "Intel DPC++" in the "Supported Compiler" column of the tables below.
+- [oneAPI DPC++ Compiler](https://github.com/intel/llvm): Open source compiler that supports CPUs and Intel, NVIDIA, and AMD GPUs. oneAPI DPC++ Compiler will be referred to as "Open DPC++" in the "Supported Compiler" column of the tables below.
+- [hipSYCL Compiler](https://github.com/illuhad/hipSYCL): Open source compiler that supports CPUs and Intel, NVIDIA, and AMD GPUs.
```

Comment:
Nice.

* I think CPUs should be clarified as x86 CPUs. 
* Its probably worth mentioning Codeplay's oneAPI for Nvidia GPU and Codeplay's oneAPI for AMD GPU in the Intel DPC++ section.
* *hipSYCL* is now *AdaptiveCpp*.

Reply:
As discussed in another comment, the support for Codeplay's oneAPI plugins can be a separate issue/PR.

Reply:
Thank you for the suggestions!

>I think CPUs should be clarified as x86 CPUs.

If I am not mistaken, I think Open DPC++ and AdaptiveCpp support CPUs more than x86 CPUs. Although including x86 in the text does not make the sentences wrong, it would give an impression that the two compilers *only* support x86 CPUs. On the other hand, including precise CPU details for the two compilers would be too heavy and irrelevant here. Since we have x86 in the tables, I would prefer to keep the description high level here.

>*hipSYCL* is now *AdaptiveCpp*.

Addressed in the commit [fa86f36](https://github.com/oneapi-src/oneMKL/pull/458/commits/fa86f36bb227a283ebbf14c856df55ad2a6aa37c). I only changed hipSYCL in README. For [building_the_project.rst](https://github.com/oneapi-src/oneMKL/blob/develop/docs/building_the_project.rst) and [selecting_a_compiler.rst](https://github.com/oneapi-src/oneMKL/blob/develop/docs/selecting_a_compiler.rst), it might be better to change them together with the source code and after testing.

Reply:
* I don't think this projects supports non-x86 CPUs, even if hipSYCL does, and to some extent Intel's open-source LLVM project. As far as I'm aware, the ROCm ecosystem is x86, as is Intel's MKL. In areas where ARM could be supported, I doubt we're testing it.
* For your second point 👍 - I've created an issue for the build documentation, so hopefully those changes can be made as part of that.

Reply:
I agree with you, but here the description is for a general introduction of the compiler, not for the aspects of the compiler oneMKL interfaces project supports. Suppose that we add "x86":
>Supported compilers include:
>- [Intel(R) oneAPI DPC++ Compiler](https://software.intel.com/en-us/oneapi/dpc-compiler): Intel proprietary compiler that supports **x86** CPUs and Intel GPUs. ...
>- [oneAPI DPC++ Compiler](https://github.com/intel/llvm): ...
>- [AdaptiveCpp Compiler](https://github.com/AdaptiveCpp/AdaptiveCpp): Open source compiler that supports **x86** CPUs and Intel, NVIDIA, and AMD GPUs.

When users go to the link of AdaptiveCpp, they will see
>AdaptiveCpp supports CPUs (including x86, arm and other LLVM-supported architectures) as well as GPUs from Intel, NVIDIA, and AMD.

I think adding "x86" would be confusing and potentially misleading.

Reply:
If you are changing hipSYCL name, please put a note saying "previously known as hipSYCL" or something like this. All the folders/files in the src/test directories still uses hipSYCL and having two different names can cause confusion.

Reply:
Yes, I have added a note for this:

![Screenshot 2024-03-14 215308](https://github.com/oneapi-src/oneMKL/assets/91495331/120b97d3-c59f-4368-b019-bb17aa35f2a3)

I copied and pasted in the above comment using the old commit and forgot to include it. Sorry for the confusion...

Reply:
> I agree with you, but here the description is for a general introduction of the compiler, not for the aspects of the compiler oneMKL interfaces project supports. 

Thats a fair enough point. Thanks @dnhsieh-intel .